tests/api2/test_disk_get_dev_size.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from middlewared.test.integration.utils import call
+
+DISKS = list(call('device.get_disks').keys())
+TYPES = (type(None), int)
+
+
+@pytest.mark.parametrize('disk', DISKS)
+def test_get_dev_size_for(disk):
+    assert isinstance(call('disk.get_dev_size', disk), TYPES)
```

Comment:
What does this test? :(

We can make a call for sda and sda3 (or whatever ZFS partition is on boot pool device) and compare results with e.g. fdisk output